js/reference.js

```diff
@@ -2745,12 +2755,20 @@ var ERMrest = (function(module) {
                         if (column.isPseudo) {
                             if (column.isForeignKey) {
                                 values[i] = column.formatPresentation(this._linkedData[column._constraintName], {context: this._pageRef._context});
-                            } else {
+                            }
+                            else {
                                 values[i] = column.formatPresentation(this._data, { formattedValues: keyValues, context: this._pageRef._context});
                             }
-                        } else {
+                        }
+                        else if (column.type.name === ("json" || "jsonb")){
```

Comment:
As I mentioned in another comment, you don't need this check here. You don't need to call another function.